src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -211,11 +211,43 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
+        // toggleCollapse binds with clicking on both collapse panels button(button to expand all
+        // panels) and collapse panels button section (button for one panel).
+        var isCollapsePanelsButtonClicked = $(e).is($('#collapse-panels-button'));
+
+        // If expand panels button is clicked and some of panel data is not yet loaded, it will be
```

Comment:
The comments here are quite long. From what I understand, would this be better:

If expand panel button is clicked and panels have classes ajax_auto or auto-response-auto, it means that the panels do not have data loaded yet. This then loads the data and removes the classes.